README.md

```diff
@@ -10,9 +10,45 @@ Istio is an open-source service mesh that provides a uniform way to manage, conn
 
 The Istio module allows you to add Istio Operator to the Kyma runtime. Within Istio Operator, Istio Controller is responsible for installing, uninstalling, and managing Istio. For more information, read the [Istio Controller documentation](./docs/user/00-10-overview-istio-controller.md).
 
+## Install Istio Manager and Istio from the latest release
+
+### Prerequisites
+
+- Access to a Kubernetes (v1.24 or higher) cluster
+- [kubectl](https://kubernetes.io/docs/tasks/tools/)
+
+### Procedure
+
+1. Clone the project and open the `istio` folder.
```

Comment:
I don't know why we should clone the project? It is not used or am I missing something?

Reply:
We do not need since we will not be deploying the sample CR but rather apply the release CR from assets.